test/bin-sky-pages.spec.js

```diff
@@ -1,10 +1,8 @@
-/*jshint jasmine: true, node: true */
-'use strict';
-
 describe('sky-pages bin', () => {
   it('should call console.error', () => {
     spyOn(console, 'error');
     require('../bin/sky-pages');
+    /* eslint-disable no-console */
```

Comment:
I'm curious what this is?

Reply:
We have an eslint rule that doesn't allow `console.log` in the source code. I needed to disable that rule for this specific unit test.